### PR TITLE
fix(chat): show placeholder on image load error

### PR DESCRIPTION
**Pull request by AlexZakablukov**
**What this PR does** 📖
show placeholder on image load error

**Which issue(s) this PR fixes** 🔨
[AP-1378](https://satellite-im.atlassian.net/browse/AP-1378)

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by AlexZakablukov** [comment]
I need correct image for placeholder

**Comment by stavares843** [comment]
added draft label due to be waiting for the image

File: components/views/chat/message/Message.vue [hunk 1]
```diff
@@ -311,6 +317,19 @@ export default Vue.extend({
...
+        return
+      }
+
+      const response = await fetch(imageSrc)
+
+      if (!response.ok) {
+        throw new Error('Could not load image')
+      }
+
+      return await response.blob()
```
Comment (molimauro) [inline components/views/chat/message/Message.vue:331]: `return await` is reduntant, simply `return`

File: components/views/chat/message/Message.vue [hunk 2]
```diff
@@ -124,8 +124,14 @@ export default Vue.extend({
...
   async mounted() {
-    const data = await fetch(this.message.payload.url)
-    this.blob = await data.blob()
+    if (!this.message.payload?.url) {
+      return
+    }
+    try {
+      this.blob = await this.getImageBlob(this.message.payload?.url)
+    } catch (error: any) {
+      this.$Logger.log('error', error.message)
```
Comment (molimauro) [inline components/views/chat/message/Message.vue:133]: can we put the image placeholder on blob error?

Comment (molimauro) [reply components/views/chat/message/Message.vue:133]: nope just `this.blob = immage placeholder` here in catch

**Comment by molimauro** [comment]
this is the placeholder image and the text that should be used:
Schermata 2022-04-13 alle 14 19 01

Can we try it? thanks ;)

**Comment by molimauro** [comment]
Minor changes: "I would make the text color the lightest text color instead of white" and change text to "Image failed to load". Thanks!

**Comment by AlexZakablukov** [comment]
done

**Review by molimauro — APPROVED** [review]

**Comment by molimauro** [comment]
To test it, just run it locally and on `components/ui/Chat/Image/Image.html` change the image url with an invalid url 🔨

**Comment by stavares843** [comment]
thanks @molimauro 🔨

**Review by stavares843 — APPROVED** [review]

**Comment by stavares843** [comment]
image

neat 🎉

**Pull request merged**